### PR TITLE
merging dev to master

### PR DESCRIPTION
**Pull request by SamyukthaSanthosh**
this is my pull request to merge dev to master

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@ <h2>Large</h2>
 <div class='various'>
     <span>World</span>
     <span>U.S. </span>
-    <span>Technology</span>
+    <span>Technologies</span>
     <span>Design</span>
```
Comment (nivendha) [inline index.html:10]: please add sports also

Comment (SamyukthaSanthosh) [reply index.html:10]: ok

Comment (nivendha) [reply index.html:10]: fine done

**Review by nivendha — APPROVED** [review]
all looks good to me , merge it

**Pull request merged**